doc/man/Makefile.am

```diff
@@ -190,6 +190,7 @@ man_MANS                = ocf_heartbeat_AoEtarget.7 \
                           ocf_heartbeat_portblock.7 \
                           ocf_heartbeat_postfix.7 \
                           ocf_heartbeat_pound.7 \
+			  ocf_heartbeat_powervs-subnet.7 \
```

Comment:
You need to use the same indentation as the other lines here.

Reply:
Finally fixed the indentation.